apps/nextjs-website/src/components/molecules/Chat/Chat.tsx

```diff
@@ -50,6 +53,7 @@ const Chat = ({
   ) satisfies Message[];
 
   const scrollRef = useRef<HTMLDivElement>(null);
+
```

Comment:
I think you should replace `backgroundColor: 'white',` with `backgroundColor: palette.background.paper,` also on line 70